CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Fixes
 
 * [#546](https://github.com/ruby-grape/grape-swagger/pull/546): Move development dependencies to Gemfile - [@olleolleolle](https://github.com/olleolleolle).
+* [#547](https://github.com/ruby-grape/grape-swagger/pull/547): Use entity_name event if type come from a string - [@frodrigo](https://github.com/frodrigo).
 
```

Comment:
The problem with the lines is that they have a period before the `-` towards the end.